common/ruletypes.h

```diff
@@ -430,6 +430,8 @@ RULE_BOOL(Spells, AlwaysSendTargetsBuffs, false, "Ignore Leadership Alternate Ab
 RULE_BOOL(Spells, FlatItemExtraSpellAmt, false, "Allow SpellDmg stat to affect all spells, regardless of cast time/cooldown/etc")
 RULE_BOOL(Spells, IgnoreSpellDmgLvlRestriction, false, "Ignore the 5 level spread on applying SpellDmg")
 RULE_BOOL(Spells, ItemExtraSpellAmtCalcAsPercent, false, "Apply the Item stats Spell Dmg and Heal Amount as Percentage-based modifiers instead of flat additions")
+RULE_REAL(Spells, BreakFeignDeathWhenCastOn, 50.0, "Percentage that fd will break when you resist a spell")
```

Comment:
`Feign Death`.

Reply:
Looks like this should default to `2.0`? Code doesn't seem to use `50.0` before introducing this.